notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def kl_sampling(theta, q, samples=100000):  # loss function when KL(p:q)
</pre><p>use <code>params</code> instead of <code>theta</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>